channeldb/waitingproof.go

```diff
@@ -172,6 +175,8 @@ func (s *WaitingProofStore) Get(key WaitingProofKey) (*WaitingProof, error) {
 
 		r := bytes.NewReader(v)
 		return proof.Decode(r)
+	}, func() {
+		proof = &WaitingProof{}
```

Comment:
nit: preexisting, but should probably be set `nil`

Reply:
Maybe better not to change it if its not broke ;P

Reply:
:beers: